docs/bugs.json

```
@@ -1,4 +1,14 @@
 [
+    {
+        "uid": "SOL-2023-3",
+        "name": "VerbatimInvalidDeduplication",
+        "summary": "``verbatim`` calls with different data can wrongfully be de-duplicated if used in very similar context.",
```

Sounds ok (other than it should be "wrongly", not "wrongfully"), but maybe that would be simpler to understand to users?

```suggestion
        "summary": "All ``verbatim`` blocks are considered identical by deduplicator and can wrongly be unified when surrounded by identical opcodes.",
```

---

I guess "incorrectly" is better?